ci_framework/roles/dlrn_report/tasks/dlrn_vars_setup.yml

```diff
@@ -0,0 +1,19 @@
+---
+- name: Get hash for job
+  ansible.builtin.include_role:
+    name: repo_setup
+    tasks_from: artifacts.yml
+
+- name: Read md5
+  ansible.builtin.command:
+    cmd: >-
+      cat {{ cifmw_repo_setup_basedir }}/artifacts/repositories/delorean.repo.md5
+  register: cifmw_dlrn_report_md5_hash
+
+- name: Set fact - full_hash
+  ansible.builtin.set_fact:
+    # Do these vars need to start with cifmw_dlrn_report_?
+    full_hash: "{{ cifmw_dlrn_report_md5_hash.stdout }}"
```

Comment:
I have improved the repo-setup ansible role via [this pr ](https://github.com/openstack-k8s-operators/ci-framework/pull/428) . It will return full hash via ` cifmw_repo_setup_full_hash` var. We can directly use it instead of reading the delorean.md5 hash file.